data/adapters/dbSNFP_adapter.py

```diff
@@ -156,45 +157,42 @@ def long_data(pos):
                         'type': 'protein coding' if (long_data(11) != -1) else 'splicing',
                         'name': 'variant of',
                         'inverse_name': 'has variant',
-                        'source': 'dbSNFP 4.5a',
+                        'source': 'dbSNFP 5.1a',
                         'source_url': 'http://database.liulab.science/dbNSFP'
                     }
                 else:
                     to_json = {
                         '_key': key,
                         'name': key,
                         'ref': data(4),
-                        'alt': data(5),
-                        'aapos': long_data(11),  # 1-based
-                        'gene_name': data(12),
+                        'alt': alt,
```

Comment:
want to load aaalt data(5)

Reply:
even if it's 'X'? currently we will replace by * otherwise the value of data(5)

Reply:
oh it's correct then, replace by * if X